isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp

```diff
@@ -997,21 +1067,30 @@ namespace Isis {
     // Pad each element in the table with the space for the longest image
     // path/name then padd it the length of the element + 1
     QString header("Measures                            RMS(pixels)");
+    if (m_statisticsResults->outputLidarData()) {
+      header += "                                   Lidar RMS(pixels)";
+    }
     // This is padded by an extra 11 to move it center to the table
     sprintf(buf,"%*s\n", header.length() + 11 + filePadding, header.toLatin1().data());
     fpOut << buf;
 
     QString dividers("***************************   *******************************************");
+    if (m_statisticsResults->outputLidarData()) {
+      dividers += "   *******************************************";
+    }
     sprintf(buf,"%*s\n", dividers.length() + 1 + filePadding, dividers.toLatin1().data());
     fpOut << buf;
 
     QString fields("|  Accepted  |   Total    |   |   Samples   |    Lines    |    Total    |");
+    if (m_statisticsResults->outputLidarData()) {
+      fields += "   |   Samples   |    Lines    |    Total    |";
+    }
```

Comment:
Why would you need a second set of Samples | Lines | Total headers?

Reply:
This table is the measure RMS error for each image. The regular measure error and the LiDAR error are split into two groups. I can see an argument for just putting them together but this is how it was in the original so I just kept that same split.

Reply:
I was thinking that the point names would divide the regular measures and the LiDAR measures, but I don't think harms having a more obvious split.